### DISCUSSION TITLE
Documentation or Help

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2563] [General]
I 've just tested this wonderful project and I am just curious how it is so good at generating realistic image generation.
Could you help me have a general idea on this project such as documentation or help or something?
I would like to learn about stable diffusion, midjourney, huggingface as well as their relationships? And the SDXL, and so on?
Even I don't know what is additionally needed to know?

If anyone can help me, Appreciate it. Thank you very much.

**Comment by mashb1t** [comment]
> "I've just tested this wonderful project"

You've already discovered this project a month ago and this is your fourth discussion ^^
It's SDXL, so feel free to research about SDXL in general.

Sources to start with:
- https://stable-diffusion-art.com/fooocus/
- https://stable-diffusion-art.com/beginners-guide/
- https://civitai.com/articles/3536/midjourney-vs-dall-e-3-vs-sdxl-10-image-comparison

Also check out the pinned / official discussions https://github.com/lllyasviel/Fooocus/discussions?discussions_q=is%3Aopen+label%3AOfficial

**Reply by antoniormrzz** [reply]
I think it would be very helpful to create an editable documentation for focus as part of this or some other repo, and give information about all tools, specially new updates. Personally I have no idea what half of the stuff in advanced is supposed to do. Searching for stuff all over the place isn't a reliable method of learning, and updates sometimes drastically change stuff. Such documentation can be created and maintained by people not working actively on the project, so that it doesn't take much of your valuable time.

**Reply by mashb1t** [reply]
I don't support the idea. While i get your intentions,the single source of truth for official documentation of Fooocus is this repository on Github. Feel free to create a wiki page, but please refer to official instructions and tutorials in either [announcements](https://github.com/lllyasviel/Fooocus/discussions/categories/announcements) or with [label official](https://github.com/lllyasviel/Fooocus/discussions?discussions_q=label%3AOfficial)
